src/components/common/Layout/Layout.tsx

```diff
@@ -78,6 +83,7 @@ const Layout = ({
         <title>{`${ogTitle ?? title}`}</title>
         <meta property="og:title" content={`${ogTitle ?? title}`} />
         <link rel="shortcut icon" href={imageUrl ?? "/guild-icon.png"} />
+        <meta httpEquiv="Cache-Control" content="max-age=31536000, public" />
```

Comment:
Do we actually need this here? It'll set the `Cache-Control` on all of our pages (where we use the `Layout` component), I think that wasn't intentional?